topics/ecology/tutorials/ENA_Biodiv_submission/tutorial.md

```diff
@@ -0,0 +1,368 @@
+
+# How to use ENA upload Tool
+
+## Adding ENA "Webin" credentials to your Galaxy user information
+
+***TODO***: *Having an ENA Submission Account: Make sure you have a submission account with the European Nucleotide Archive (ENA). You will need the identifier and the password.*
+
+>  <hands-on-title> Add your "WEBIN" credentials to your Galaxy account </hands-on-title>
+>  **Instructions:**
+>	- From the Menu, click on "User" > "Preferences". Click on "Manage Information". Scroll down to "Your ENA Webin account details" and enter your ENA "Webin" identifier and password.
+>  ![Adding ENA Webin credentials](./images/imageI.png)
+{: .hands_on}
+
+## Submitting using a metadata template file
+
+For this tutorial we will use the ENA default sample checklist.
+
+![Excel Metadata template](./images/image2.png)
+
+**Note:** It is crucial to fill in all the fields marked "Mandatory" and ensure that the sequence names match exactly those indicated in the Excel file.
+
+> <comment-title> ENA Metadata Templates </comment-title>
+>
+> You can find metadata templates for each checklist in the [ELIXIR-Belgium GitHub repository](https://github.com/ELIXIR-Belgium/ENA-metadata-templates)
+>
+> 1. Direct download link of the [ENA default sample checklist]( https://github.com/ELIXIR-Belgium/ENA-metadata-templates/raw/main/templates/ERC000011/metadata_template_ERC000011.xlsx)
+>
+> 2. Direct download link of the [ENA default sample checklist filled with elements fot the training](https://github.com/galaxyproject/training-material/raw/24776cf161e38ac0449755749d23e851400020aa/topics/ecology/tutorials/ENA_Biodiv_submission/metadata_GdBqCOI_ERC000011_Test.xlsx)
+>
+> You will need to import this file into your Galaxy history. Then, use the ENA Upload Tool to proceed with the submission.
+>
+{: .comment}
+
+
+
+> <hands-on-title> Excel Metadata Template </hands-on-title>
+>
+> 1. Import the ENA default sample checklist file.
+>
+>    ```
+>    https://github.com/galaxyproject/training-material/raw/24776cf161e38ac0449755749d23e851400020aa/topics/ecology/tutorials/ENA_Biodiv_submission/metadata_GdBqCOI_ERC000011_Test.xlsx
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 2. {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/multiqc/multiqc/1.11+galaxy1) %} with the following parameters:
+>    - *"Action to execute"*: `Add new (meta)data`
+>    - *"Select the metadata input method"*: `Excel file`
+>    - *"Select the ENA sample checklist"*: `ENA default sample checklist (ERC000011)`
+>    - *"Select input data"*: `Dataset or dataset collection`
+>
+>    > <comment-title> Datatype </comment-title>
+>    >
+>    > The ENA upload tool only allows .fastq.gz files for submission
+>    {: .comment}
+>
+>    > <warning-title> Danger: Submit to ENA test server! </warning-title>
+>    > We suggest you first submit to the test server before making a public submission!
+>    {: .warning}
+>
+> ![ENA Upload tool](./images/image3.png)

```

Comment:
we try to stay away from screenshots as much as possible, since the Galaxy interface changes too often. Please make sure all parameters are captured in text form above and remove this screenshot

Reply:
Added !

Reply:
Thanks! Let's remove the screenshot then :)

```suggestion
```